Sources/MarathonCore/URL+Marathon.swift

```diff
@@ -55,4 +55,23 @@ internal extension URL {
 
         return URL(string: parentString).require()
     }
+    
+    func transformIfNeeded() -> URL {
+        guard isGithubURL else { return self }
```

Comment:
Could you put `return self` on its own line. This might seem like a style thing, but it's also very useful when measuring test coverage, since the return will be matched separately 🙂